clustered/common/src/test/java/org/ehcache/clustered/common/ServerStoreCompatibilityTest.java

```diff
@@ -0,0 +1,485 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.clustered.common;
+
+import junit.framework.Assert;
+import org.ehcache.clustered.common.ServerStoreConfiguration.PoolAllocation;
+import org.ehcache.clustered.common.ServerStoreConfiguration.PoolAllocation.Fixed;
+import org.ehcache.clustered.common.ServerStoreConfiguration.PoolAllocation.Shared;
+import static org.junit.Assert.fail;
+import org.junit.Test;
+
+/**
+ *
+ * @author GGIB
+ */
+public class ServerStoreCompatibilityTest
```

Comment:
You should observe the existing code formatting. There is some freedom but not around curly braces placement.